src/ssl.c

```diff
@@ -34269,6 +34291,24 @@ word32 nid2oid(int nid, int grp)
             }
             break;
 
+        /* oidCmsKeyAgreeType */
+    #ifdef WOLFSSL_CERT_REQ
+        case oidCsrAttrType:
+            switch (nid) {
+                case NID_pkcs9_contentType:
+                    return PKCS9_CONTENT_TYPE_OID;
+                case NID_pkcs9_challengePassword:
+                    return CHALLENGE_PASSWORD_OID;
+                case NID_serialNumber:
+                    return SERIAL_NUMBER_OID;
+                case NID_userId:
+                    return USER_ID_OID;
+                case NID_surname:
+                    return SURNAME_OID;
```

Comment:
handling of default case where nid is not an expected value

Reply:
There already is a default case below

Reply:
I see a default case for the surrounding switch statement but not the inner switch statement on 'nid'. Looking at it again though if it hits the break case it will go to the end of the function and return (word32)-1 ....